docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,191 @@
+
+`DataGrid` and `DataGridPro` now feature a brand-new virtualization engine!
+We decided to rewrite it completely to address the many issues raised by the community, and to make easier to release new features that impact the rendering of rows and columns.
+An advantage over the previous version is that we now use the native scroll.
+This means that scrolling the grid is like scrolling a webpage,
+so any jittering caused when the scroll is overridden is gone.
+
+Talking about performance, one of the main problems we had was that scrolling horizontally was laggy compared to vertical scrolling.
+After investigation we found that, although the columns were virtualized, a lot of unnecessary renders were occurring.
+This can be seen in the top part of the screenshot below, where it compares `v5.0.0-beta.4` (the last version before the new virtualization engine) with `v5.0.0`.
+On each scroll event it renders again, and each frame takes a long time to be drawn (some are even lost).
+To address these problems, we took the following actions:
+
+- Avoid rendering the entire grid during scroll.
+- Pass the correct value to the `key` prop to ensure that React will reuse the existing DOM nodes.
+- Reduced the number of event listeners attached to each cell.
+- Increased the number of columns rendered in the overscan (the extra columns rendered to make scroll smoother).
+- Delay the rendering of new columns whenever possible.
+
+The result of these changes is shown in the bottom part of the comparison.
+The number of frames that could be drawn in the same amount of time dramatically increased, compared to the previous virtualization approach.
+In our [benchmark](https://github.com/mui-org/material-ui-x/pull/2673), the FPS (frames per second) went from 22 to 42, on average.
+The time each frame takes to be rendered, indicated by the width of each block, was reduced.
+
+The idea of delaying the re-rendering also can be seen in the large voids between the blocks.
+Each void means that a re-render was not necessary since the required columns were already rendered by the overscan.
+
+![Performance comparison between v5.0.0-beta.4 and v5.0.0](https://user-images.githubusercontent.com/42154031/141475697-11281f83-9a2f-4a2b-8001-f459cbdccbd8.png)
+
+Some of the mentioned improvements were also applied to the rows, however the gains were more subtle.
+Besides the better performance, the new virtualization engine also brings the following fixes:
+
+- Horizontal and vertical scroll share the same logic.
+- No more jumps when changing the rendered rows.
+- Calling `apiRef.current.scrollToIndexes` works no matter where the cell is.
+- Improved support for when `disableVirtualization` is used.
+- Fixes keyboard navigation with arrow keys.
+
+## Improved state management
+
+Several enhancements were made to state management to improve developer experience, performance and consistency in the execution order.
+
+#### Improving the developer experience around the state
```

Comment:
this is an h4, the parent is an h2. This is both 1. wrong for accessibility (added to https://www.notion.so/mui-org/Technical-documentation-7e55b517ac2e489a9ddb6d0f6dd765de) and 2. breaking the TOC.

x the other instances

Reply:
Also, we could shorten the header, it's long, the tense is different from the other headers, "around" is vague, we could make it more concrete:
```suggestion
### Improved DX when using the state
```